app/assets/javascripts/topbar.js

```diff
@@ -1,16 +1,9 @@
 $(function() {
-  var clone = $('.persist-header').clone();
-  $('.persist-header').after(clone);
-  $('.persist-header:first').hide();
-  $('.persist-header:last').css({
-    'position': 'static', 'top': '0'
-  });
-
   mark_active_menu();
 
   $('.dropdown-toggle').dropdown();
   if(!is_mobile()){
-    $(window).on('scroll', function() {onScroll(this)});
+    $(window).on('scroll', persistHeader);
```

Comment:
`this` in this context is the window, no need to pass it as a parameter. Also, function name should be clearer.